packages/e2e/lib/common.dart

```diff
@@ -11,13 +11,20 @@ class Response {
 
   final bool _allTestsPassed;
 
+  /// The extra information to be added along side the test result.
+  ///
+  /// Specially `'result'` and `'failureDetails'` keys are occupied, and will
+  /// be ignored.
+  Map<String, dynamic> extraInfo;
+
   /// Constructor to use for positive response.
-  Response.allTestsPassed()
+  Response.allTestsPassed([this.extraInfo])
```

Comment:
Prefer optional named parameters to optional position parameters. They get really hard to think about as you add more :)

Reply:
Done!